azerothcore/azerothcore-wotlk
# Fix issue 6053 laughing sisters faction

## Changes Proposed:
Correct behavior should have mobs friendly to Ally, hostile to Horde.
Correct behavior should also have faction Darnassus and not flag PVP.
All of these are met with faction 124, which is used for similar mobs in Stonetalon Peak.

## Issues Addressed:
Flagging players PVP and alerting PVP channel of an attack.
- Closes #6053  https://github.com/azerothcore/azerothcore-wotlk/issues/6053

## SOURCE:
This is how it is on classic atm (and always has been, the incorrect faction is an artifact from data mined from wowhead most likely, where the other darnassus factions do not exist)

## Tests Performed:
 


## How to Test the Changes:


1. create ally or horde character 
2. .go c 34199 and observe intraction (friendly Darnasuss to ally, hostile to horde, does not flag for pvp interaction) 
3. repeat on other faction if unsure of functionality still

## Known Issues and TODO List:

Issue should be resolved


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
